backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlRawQueryForSchema.java

```diff
@@ -20,26 +26,59 @@ public SqlRawQueryForSchema(SqlSchema schema) {
     this.schema = schema;
   }
 
-  public List<Row> executeSql(String sql) {
+  public List<Row> executeSql(String sql, Map<String, ?> parameters) {
     if (sql == null || sql.trim().equals("")) {
```

Comment:
how come the value is a ?, is it not so that you have a finite amount of types of the parameter values? Might want to create a class for that

Reply:
no, we don't know beforehand what type it is. So ? is perfect. Even Object would limit it (I tried). Wontfix.